src/ign.hh

```diff
@@ -121,6 +121,7 @@ extern "C" IGNITION_FUEL_TOOLS_VISIBLE int pbtxt2Config(const char *_path);
 /// \return 1 if successful, 0 if not.
 extern "C" IGNITION_FUEL_TOOLS_VISIBLE int editUrl(
     const char *_url, const char *_header = nullptr,
-    const char *_private = nullptr);
+    const char *_private = nullptr,
+    const char *_path = nullptr);
```

Comment:
Missing doxygen

Reply:
Added in e3ae7d852192597e06022dbb5a3e9a03cdca5485